src/main/scala/stdlib/CaseClasses.scala

```diff
@@ -71,8 +71,8 @@ object CaseClasses extends AnyFlatSpec with Matchers with org.scalaexercises.def
    * It only makes sense to define case classes if pattern matching is used to decompose data structures. The following object defines a pretty printer function for our lambda calculus representation:
    *
    * {{{
-   * object TermTest extends Application {
```

Comment:
I think this was originally meant as `extends App` :+1:

Reply:
Right, okay. Fixed it and updated the pull request